recipes/csm/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "libcsm" %}
+{% set version = "3.0.3" %}
+{% set commit = "9b11afac782fe210003310c9868af8480ab05c9c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/csm/archive/{{ commit }}.tar.gz
+  sha256: ad541f22fb491c93d9d4fe08f7650c4bf6b815f0354212b12a4c62d4f12365ea
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - test -e $PREFIX/lib/libcsmapi.so  # [linux]
+    - test -e $PREFIX/lib/libcsmapi.dylib  # [osx]
+    - test -e $PREFIX/include/csm/Isd.h
+    - test -e $PREFIX/include/csm/csm.h
+
+about:
+  home: https://github.com/USGS-Astrogeology/csm
+  license: None
```

Comment:
It looks as if the software is licensed under ["The Unlicense"/ the public domain](https://github.com/USGS-Astrogeology/csm/blob/master/LICENSE)

Reply:
Ah thanks for catching that